CHANGELOG.md

```diff
@@ -6,6 +6,39 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) a
 
 ## [Unreleased]
 
+### Adds
+
+- *BREAKING*: Update SqlServerDsc to 11.0.0.0 ([MODULES-6592](https://tickets.puppetlabs.com/browse/MODULES-6592))
```

Comment:
Nit: We're currently wrapping this as **BREAKING**, the line here will be emphasized instead of strong.